src/components/ui/IconButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {
+  ButtonHTMLAttributes,
+  PropsWithChildren,
+  forwardRef,
+  ForwardedRef,
+  useId,
+} from 'react'
+import { Button, Icon } from '@/components/index'
+import clsx from 'clsx'
+
+export interface IconButtonProps
+  extends ButtonHTMLAttributes<HTMLButtonElement> {
+  label: any
```

Comment:
```suggestion
  label: ReactNode
```

en de import toevoegen

Reply:
✅